src/Platform/Microsoft.Testing.Platform/OutputDevice/Terminal/TerminalTestReporter.cs

```diff
@@ -108,10 +108,12 @@ private static Regex GetFrameRegex()
     }
 #endif
 
+    private int _counter;
+
     /// <summary>
     /// Initializes a new instance of the <see cref="TerminalTestReporter"/> class with custom terminal and manual refresh for testing.
     /// </summary>
-    internal TerminalTestReporter(IConsole console, TerminalTestReporterOptions options)
+    public TerminalTestReporter(IConsole console, TerminalTestReporterOptions options)
```

Comment:
Please add option for showing test details so we can disable this in `dotnet test` until it is ready for multi project runs.

Reply:
added the option to TerminalTestReporterOptions